proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -466,10 +466,40 @@ message MergeDLQTasksRequest {
 }
 
 message MergeDLQTasksResponse {
-  // TODO: add DescribeDLQJob an CancelDLQJob APIs for clients to use these tokens.
   bytes job_token = 1;
 }
 
+message DescribeDLQJobRequest {
+  string job_id = 1;
```

Comment:
Should be job_token to be consistent with the response above.

Reply:
done